packages/govern-console/src/containers/CreateDAO/CreateDAO.tsx

```diff
@@ -129,281 +146,84 @@ const SubTitle = styled(Typography)({
 const NewDaoForm: React.FC<FormProps> = memo(
   ({ setCreateDaoStatus, setCreatedDaoRoute, cancelForm }) => {
     const context: any = useWallet();
+    const { chainId, status, ethersProvider } = context;
+
     const {
-      // connector,
-      // account,
-      // balance,
-      chainId,
-      // connect,
-      // connectors,
-      // ethereum,
-      // error,
-      // getBlockNumber,
-      // networkName,
-      // reset,
-      status,
-      // type,
-      ethersProvider,
-    } = context;
-
-    const _chainId = useMemo(() => {
-      if (chainId === 4 && status === 'connected') {
-        return 4;
+      control,
+      handleSubmit,
+      watch,
+      setValue,
+      getValues,
+    } = useForm<FormInputs>();
+
+    const connectedChainId = useMemo(() => {
+      if (chainId === ChainId.RINKEBY && status === 'connected') {
+        return ChainId.RINKEBY;
```

Comment:
@Rekard0 

There's no need to have `else` statement. Directly doing `return ChainId.MAINNET;` should be enough. But i am not sure if `status` is not connected, why we are returning `ChainId.MAINNET;`